assets/panel/debugger.properties

```diff
@@ -597,7 +597,7 @@ sources.search=%S to search
 
 # LOCALIZATION NOTE (watchExpressions.header): Watch Expressions right sidebar
 # pane header.
-watchExpressions.header=Watch expressions
+watchExpressions.header=Watch Expressions
```

Comment:
Is title case expected here? There's at least another header that doesn't use it, but I'm not sure if the context is different

Reply:
This shouldn't change -- will remove before merging @flodolo